crates/core/src/worker_metrics/mod.rs

```diff
@@ -85,6 +85,11 @@ metrics_group!(
         #[labels(caller_identity: Identity, module_hash: Hash, caller_address: Address, reducer_symbol: str)]
         pub wasm_instance_errors: IntCounterVec,
 
+        #[name = spacetime_worker_wasm_allocated_memory]
+        #[help = "The number of bytes of linear memory allocated by the database's WASM module instance"]
+        #[labels(database_identity: Identity)]
+        pub wasm_allocated_memory: IntGaugeVec,
```

Comment:
I think `wasm_memory_bytes` would be a bit more clear, since the amount of memory allocated is usually not the same as the amount of memory used.

Reply:
Well, this is the amount of memory allocated, but it's still taking up RAM whether or not the wasm module is actually using it or not.